nrf_security/src/drivers/nrf_oberon/CMakeLists.txt

```diff
@@ -35,6 +35,9 @@ if (COMPILE_PSA_APIS)
     oberon_rsa.c
     oberon_ctr_drbg.c
     oberon_hmac_drbg.c
+    oberon_jpake.c
+    oberon_srp.c
+    oberon_spake2p.c
```

Comment:
These three needs to excluded when not enabled. Since they probably won't compile when PSA_CORE_BUILTIN is enabled. The same issue already exists for oberon_ctr/hmac_drbg.

The macro append_with_prefix_ifdef is broken, but after this PR is merged you can use that one:
https://github.com/nrfconnect/sdk-nrfxlib/pull/971/files